src/Common/AbstractProcessor.php

```diff
@@ -88,9 +85,8 @@ public function handleException(Exception $exception, $level = 'warning', array
         $title = 'Handle Exception: '. $level;
         $messageException = $exception->getMessage();
         $message = $this->formatNotification->formatMessage($title,false,$messageException);
-        $this->logger->$level(
-            $message, $context
-        );
+        $this->logger->$level($message, $context);
+        
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1131)